assets/app/scripts/directives/quotaUsageChart.js

```diff
@@ -56,7 +56,13 @@ angular.module('openshiftConsole')
         }
 
         // https://github.com/mbostock/d3/wiki/Formatting
-        var percentage = d3.format(".2p");
+        var percentage = function(value) {
+          if (!value) {
+            return "0%";
+          }
+
+          return (value.toFixed(3) * 100) + "%";
```

Comment:
are you guaranteed value is a number? do you need to parse it or anything to be sure